lib/modules/dosomething/dosomething_campaign/dosomething_campaign.module

```diff
@@ -843,14 +839,22 @@ function dosomething_campaign_is_active($node) {
    *
    * @param DateTime $date
    *  DateTime to edit
+   * @param boolean $add_four
+   *  Optional - prevents default EST datetime conversion from becoming previous day
    * @param boolean $end_of_day;
    *  Optional - set the date to end of day
    *
    * @return DateTime
    *  Updated time
    */
-function dosomething_campaign_set_est_timezone($date) {
+function dosomething_campaign_set_est_timezone($date, $add_four = FALSE, $end_of_day = FALSE) {
+  if ($add_four) {
+    $date->add(new DateInterval('PT4H'));
```

Comment:
looks like I should be able to use this function,

http://php.net/manual/en/datetimezone.getoffset.php

something along the lines of,
```
if ($add_four) {
  $user_timezone 
  $difference = (new DateTime().getOffsest($user_timezone) / 60) / 60;
  $date->add(new DateInterval('PT' . $difference . 'H'));
}
```